### ISSUE TITLE
Installing ponyd fails due to pybonjour download failure

### ISSUE THREAD

**Issue by mark-anders** [square/PonyDebugger#126]
I'm using OS X 10.10 with the latest tools installed.  When using the curl method to install ponyd and removing the -k parameter as per #125, I get the following error:
```
Downloading/unpacking pybonjour (from ponydebugger)
  Could not find any downloads that satisfy the requirement pybonjour (from ponydebugger)
  Some externally hosted files were ignored (use --allow-external pybonjour to allow).
Cleaning up...
No distributions at all found for pybonjour (from ponydebugger)
```
I've checked the file ~/Library/PonyDebugger/src/ponydebugger/scripts/_bootstrap_contents.py and it contains '--allow-external'.  

Any way to work around it?

**Comment by zettlunic** [comment]
Thank you very much @mark-anders!

**Comment by zkirill** [comment]
Thank you @mark-anders!

**Comment by codesourse** [comment]
when i use "pip install -U -e git+https://github.com/square/PonyDebugger.git#egg=ponydebugger --allow-external pybonjour --allow-unverified pybonjour"
i meet following problem ,anyone can help me.

Collecting pybonjour (from ponydebugger)
  Retrying (Retry(total=4, connect=None, read=None, redirect=None)) after connection broken by 'ProtocolError('Connection aborted.', error(65, 'No route to host'))': /p/pybonjour/
  Retrying (Retry(total=4, connect=None, read=None, redirect=None)) after connection broken by 'ProtocolError('Connection aborted.', error(65, 'No route to host'))': /p/pybonjour/
  Retrying (Retry(total=3, connect=None, read=None, redirect=None)) after connection broken by 'ProtocolError('Connection aborted.', error(65, 'No route to host'))': /p/pybonjour/

**Comment by FromPointer** [comment]
@codesourse,  i have the same problem. how to solve it?

**Comment by ybbaigo** [comment]
+1 for the same issue as @codesourse

**Comment by iamdaiyuan** [comment]
thank you.

**Comment by codesourse** [comment]
use this log  https://github.com/fpillet/NSLogger

**Comment by Kirow** [comment]
Thank you @mark-anders!

**Comment by zziking** [comment]
@codesourse @ybbaigo is because of your host can not reach the pybonjour code in google's server:  https://code.google.com/p/pybonjour/downloads/list?can=1, you can use --proxy option for pip install, such as : 
pip install -U -e git+https://github.com/square/PonyDebugger.git#egg=ponydebugger --allow-external pybonjour --allow-unverified pybonjour --proxy="the proxy can reach code.google.com"

**Comment by grioja** [comment]
Is there an update here? Pretty much any installation on Mac simply fails.

**Comment by Derimagia** [comment]
Pretty sure this is related to / a duplicate of Bug #100

**Comment by justinseanmartin** [comment]
I believe this should be addressed by #186. If not, please use issue #100 to track this.
